website/src/components/Chat/PluginsChooser.tsx

```diff
@@ -130,8 +130,8 @@ export const PluginsChooser = ({ plugins }: { plugins: PluginEntry[] }) => {
             </Box>
           )}
         </MenuButton>
-        <Box position="fixed" zIndex="1">
-          <MenuList left="-150px" position="absolute" w="280px">
+        <Box position="fixed" zIndex="2">
+          <MenuList left="-150px" position="absolute" w="280px" h="400px" overflowY="auto">
```

Comment:
Can you revert this? Already fixed in my PR

Reply:
are plugins actually released yet since i cant select any in the chat window

Reply:
Yes, plugins are working in prod. You would need to past URL of some plugins, you could check out inference/worker/plugins.md for more info

Reply:
we could put a link to the docs into the frontend

Reply:
thanks having some problems with wolfram alpha though it always gives errors back when the assistant trys to use it  any tips?

Reply:
I will update plugin doc, a bit, and leave frontend update to notmd(#3050), he already will be including some labels and hover on descriptions fot plugins.

Reply:
> thanks having some problems with wolfram alpha though it always gives errors back when the assistant trys to use it any tips?

wolfram is behind auth, and only available for openai atm, we are waiting from them to update cloud llm api, so we get api key from them.

Reply:
ah i see also i saw you demo the web search and calculator plugin is there currently any way to use those?

Reply:
Unfortunately, those plugins were local only for my dev and testing, but here you can checkout my boilerplate plugin.
https://github.com/draganjovanovich/web-retriever
and its hosted on this URL: https://web-retriever-draganjovanovich.vercel.app/ai-plugin.json

Reply:
Yeah, first thing that would help immensely is ctx size increase.
btw, you could check out some of non authenticated plugins here:
https://www.getit.ai/gpt-plugins
And also on OA discord server, I posted quite a few cool ones, and also there are also a couple of OA community created already :))